docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/prepare-environment/prepare-network.md

```diff
@@ -0,0 +1,896 @@
+   <Tabs>
+
+   <TabItem label="Inbound Rules" value="inbound-rules">
+
+   | Protocol    | Port | Source                           | Description                                                     |
+   | ----------- | ---- | -------------------------------- | --------------------------------------------------------------- |
+   | UDP         | 1194 | `52.44.108.101`, `3.225.148.144` | OpenVPN traffic.                                                |
+   | UDP         | 500  | `52.44.108.101`, `3.225.148.144` | IKE Phase 1 for IPsec VPN.                                      |
+   | UDP         | 4500 | `52.44.108.101`, `3.225.148.144` | NAT-Traversal (NAT-T) for IPsec VPN.                            |
+   | ESP (IP 50) | N/A  | `52.44.108.101`, `3.225.148.144` | Encapsulating Security Payload (ESP) for IPsec data encryption. |
+
+   </TabItem>
+
+   <TabItem label="Outbound Rules" value="outbound-rules">
+
+   | Protocol    | Port | Destination                      | Description                                                     |
+   | ----------- | ---- | -------------------------------- | --------------------------------------------------------------- |
+   | UDP         | 1194 | `52.44.108.101`, `3.225.148.144` | OpenVPN traffic.                                                |
+   | UDP         | 500  | `52.44.108.101`, `3.225.148.144` | IKE Phase 1 for IPsec VPN.                                      |
+   | UDP         | 4500 | `52.44.108.101`, `3.225.148.144` | NAT-Traversal (NAT-T) for IPsec VPN.                            |
+   | ESP (IP 50) | N/A  | `52.44.108.101`, `3.225.148.144` | Encapsulating Security Payload (ESP) for IPsec data encryption. |
+
+   </TabItem>
+
+   </Tabs>
+
+9. Ensure that the appropriate NAT exemptions or policies, such as IPsec passthrough, are configured so that IPsec
+   traffic is not inadvertently translated.
+
+10. Configure your on-prem/remote router to ensure network traffic to and from AWS reaches the correct hybrid nodes.
+
+    In both BGP and static routing scenarios, a route must exist to send all Amazon EKS VPC-bound traffic through a
+    centralized VPN gateway.
+
+    <Tabs>
+
+    <TabItem label="Border Gateway Protocol (BGP)" value="bgp">
+
+    - Configure BGP on your on-prem/remote router to establish dynamic routing with AWS.
+      - AWS Site-to-Site VPN supports BGP over IPsec to automatically exchange routes between the on-prem/remote network
+        and AWS.
+    - Ensure that the on-prem/remote router is advertising its internal network to AWS and accepting AWS VPC CIDR routes
+      via BGP to direct traffic correctly.
+
+    </TabItem>
+
+    <TabItem label="Static Routing" value="static">
+
+    - Static routes are configured during the
+      [Configure Hybrid Node Networking for VPN Solutions](../create-hybrid-node-pools.md#configure-hybrid-node-networking-for-vpn-solutions)
+      steps.
+    - Optionally, you can define a unique VPN server IP for each hybrid node during the
```

Comment:
This comment doesn't necessarily apply to only static routing. The VPN server IP feature is available as a fallback in case the edge network's default gateway doesn't know to automatically route packets destined for the AWS VPC CIDR through the VPN server. If the network already knows how to do that, the VPN server IP feature adds no value.

I just want to clarify that the network's default gateway may not have such a route even if you are using BGP to advertise node CIDR ranges to AWS and/or a local router.

Reply:
Understood, I've added additional context to the BGP tab to address this.